dev/AppLifecycle/ActivationRegistrationManager.cpp

```diff
@@ -8,47 +8,89 @@
 #include "ProtocolActivatedEventArgs.h"
 #include "FileActivatedEventArgs.h"
 #include "Association.h"
+#include "ExtensionContract.h"
+#include "EncodedLaunchExecuteCommand.h"
+
+extern "C" HRESULT __stdcall DllRegisterServer() noexcept;
```

Comment:
>DllRegisterServer [](start = 29, length = 17)

Why?

Is this pre-DynamicDependencies workaround?

Reply:
This has nothing to do with DynamicDependencies.  This is purely because it is DllRegisterServer which is special and there is no reason to name a differently named function just to wrap this one.

Reply:
Ah, I think I see what you might be thinking.

Yes, this code is for a world where we don't have a framework package that we can load yet.  Until your changes make it into main, we are still pre-DynamicDependencies and thus need this.  I won't be removing it until your changes are in and my tests succeed.  So yeah, once I can actually register COM objects to be hosted inside the framework package this code must remain.

Reply:
I suspected. Or at least, I hoped so :-)

Just checking that there's not some longer-term reason we need it once DynDep, Framework package and other foundation's available